.github/workflows/main.yml

```diff
@@ -0,0 +1,16 @@
+name: Publish docker image
```

Comment:
The generator currently has Travis for continuous integration (CI) I'd suggest choosing one CI-system and sticking with that one.

Reply:
yeap, and I suggest github actions, they are very well integrated into Github, then we have one tool for code and its build, and super nicely integrated secrets management. This PR was a test how it works, and I have to say I'm pleased as I was able to integrate it in just few hours without any previous knowledge of the tool

Reply:
I have zero preference for one CI system over another. As long as they get the job done. However, having two CI-systems in the same code-base does make my spidey senses tingle. If you're ok with having two that's fine with me. Either way, I created a CI/CD issue today. I could add the move to GitHub actions in there to make sure its on the TODO-list? Issue -> [here](https://github.com/asyncapi/generator/issues/220)

Reply:
I saw the issue, awesome stuff. I think that having consistent flow and tooling across all repos + dev-friendly setup is something that should enable easier onboarding of new contributors to the project